specification/ServiceBusManagement.json

```diff
@@ -0,0 +1,413 @@
+        ],
+        "operationId": "Queue_Get",
+        "description": "Get the details about the queue found at the provided queuePath.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/QueuePathParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/EnrichFalseParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Get Queue operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/GetQueueResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Queues"
+        ],
+        "operationId": "Queue_Create",
+        "description": "Create a new queue at the provided queuePath",
+        "parameters": [
+          {
+            "$ref": "#/parameters/QueuePathParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CreateQueueParameters"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created -- Create Queue operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/GetQueueResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "GetQueueResponse": {
+      "description": "The response from a GetQueue operation",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The ID Of the Queue",
+          "type": "string"
+        },
+        "title": {
+          "description": "The name of the resource",
+          "type": "string"
+        },
+        "published": {
+          "description": "The timestamp for when this resource was published",
+          "type": "string"
+        },
+        "updated": {
+          "description": "The timestamp for when this resource was last updated",
+          "type": "string"
+        },
+        "author": {
+          "description": "The author that created this resource",
+          "type": "object",
+          "properties": {
+            "name": {
+              "description": "The name of the author that created this resource",
+              "type": "string"
+            }
+          }
+        },
+        "link": {
+          "description": "TODO: Add description",
+          "type": "object",
+          "properties": {
+            "rel": {
+              "description": "TODO: Add description",
+              "type": "string",
+              "xml": {
+                "attribute": true
+              }
+            },
+            "href": {
+              "description": "TODO: Add description",
+              "type": "string",
+              "xml": {
+                "attribute": true
+              }
+            }
+          }
+        },
+        "content": {
```

Comment:
This should be a top level model in "definitions" and content should have a reference to that model